packages/ethereum-contracts/test/contracts/superfluid/SuperToken.NonStandard.test.ts

```diff
@@ -92,13 +92,18 @@ describe("SuperToken's Non Standard Functions", function () {
             await expectCustomError(
                 superToken.updateCode(ZERO_ADDRESS),
                 superToken,
-                "SUPER_TOKEN_ONLY_HOST"
+                "SUPER_TOKEN_ONLY_ADMIN"
             );
         });
 
         it("#1.4 only can initialize once", async () => {
             await expectRevertedWith(
-                superToken.initialize(ZERO_ADDRESS, 18, "name", "symbol"),
+                superToken["initialize(address,uint8,string,string)"](
```

Comment:
Are we really going there? Feels a bit like a breaking change, although technically it isn't.

Reply:
yup, hard to avoid this tbh

Reply:
avoid name conflict is also an option sometimes: name the new function initializeWithAdminOverride

Reply:
fair-we can do this instead